src/data/thermal-conductivity-constants.ts

```diff
@@ -2774,6 +2774,676 @@ export const DIPPRThermalConductivityDictionary: Record<
+    minimumTemperature: 116.34,
+    maximumTemperature: 216.25,
+    thermalConductivityAtMinimumTemperature: 0.2139,
+    thermalConductivityAtMaximumTemperature: 0.1593,
+  },
+
+  alphaMethylStyrene: {
+    name: 'alpha-Methyl styrene',
+    formula: 'C9H10',
+    CAS: '98-83-9',
+    molecularWeight: 118.1757,
+    C1: 0.19657,
+    C2: -0.0002118,
+    minimumTemperature: 249.95,
+    maximumTemperature: 438.65,
+    thermalConductivityAtMinimumTemperature: 0.1436,
+    thermalConductivityAtMaximumTemperature: 0.1037,
+  },
+
+  MethylTertbutylEther: {
+    name: 'Methyl tert-butyl ether',
+    formula: 'C5H12O',
+    CAS: '1634-04-4',
+    molecularWeight: 88.1482,
+    C1: 0.22526,
+    C2: -0.00037235,
+    C3: 1.1689e-7,
+    C4: 0,
+    C5: 0,
+    minimumTemperature: 164.55,
+    maximumTemperature: 328.2,
+    thermalConductivityAtMinimumTemperature: 0.1672,
+    thermalConductivityAtMaximumTemperature: 0.1156,
+  },
+
+  MethylVinylEther: {
+    name: 'Methyl vinyl ether',
+    formula: 'C3H6O',
+    CAS: '107-25-5',
+    molecularWeight: 58.07914,
+    C1: 0.28035,
+    C2: -0.0004646,
+    minimumTemperature: 151.15,
+    maximumTemperature: 341.1,
+    thermalConductivityAtMinimumTemperature: 0.2101,
+    thermalConductivityAtMaximumTemperature: 0.1219,
+  },
+
+  Naphthalene: {
+    name: 'Naphthalene',
+    formula: 'C10H8',
+    CAS: '91-20-3',
+    molecularWeight: 128.17052,
+    C1: 0.17096,
+    C2: -0.00010059,
+    minimumTemperature: 353.43,
+    maximumTemperature: 646.97,
+    thermalConductivityAtMinimumTemperature: 0.1354,
+    thermalConductivityAtMaximumTemperature: 0.1059,
+  },
+
+  Neon: {
+    name: 'Neon',
+    formula: 'Ne',
+    CAS: '7440-01-9',
+    molecularWeight: 20.1797,
+    C1: 0.2971,
+    C2: -0.017356,
+    C3: 0.0005911,
+    C4: -0.000007421,
+    minimumTemperature: 25,
+    maximumTemperature: 44,
+    thermalConductivityAtMinimumTemperature: 0.1167,
+    thermalConductivityAtMaximumTemperature: 0.0457,
+  },
+
+  Nitroethane: {
+    name: 'Nitroethane',
+    formula: 'C2H5NO2',
+    CAS: '79-24-3',
+    molecularWeight: 75.0666,
+    C1: 0.247,
+    C2: -0.0002814,
+    minimumTemperature: 183.63,
+    maximumTemperature: 387.22,
+    thermalConductivityAtMinimumTemperature: 0.1953,
+    thermalConductivityAtMaximumTemperature: 0.138,
+  },
+
+  Nitrogen: {
+    name: 'Nitrogen',
+    formula: 'N2',
+    CAS: '7727-37-9',
+    molecularWeight: 28.0134,
+    C1: 0.2654,
+    C2: -0.001677,
+    minimumTemperature: 63.15,
+    maximumTemperature: 124,
+    thermalConductivityAtMinimumTemperature: 0.1595,
+    thermalConductivityAtMaximumTemperature: 0.0575,
+  },
+
+  NitrogenTrifluoride: {
+    name: 'Nitrogen trifluoride',
+    formula: 'F3N',
+    CAS: '7783-54-2',
+    molecularWeight: 71.00191,
+    C1: NaN,
+    C2: NaN,
+    minimumTemperature: NaN,
+    maximumTemperature: NaN,
+    thermalConductivityAtMinimumTemperature: NaN,
+    thermalConductivityAtMaximumTemperature: NaN,
```

Comment:
```suggestion
```

The interface should be adopted accordingly.

Reply:
- [ ] Also add a test case (Jest) for this particular compound.

Reply:
Was the test case added?

Reply:
yeah , I have added it ,